src/sql_queries.py

```diff
@@ -41,10 +42,46 @@
 """
 
 query_evaluation_data = """
-    SELECT
-        id,
-        ARRAY_TO_STRING(labels, ", ") as labels,
-        urgency
-    FROM
-        @EVALUATION_TABLE
-    """
+SELECT
+    id,
+    ARRAY_TO_STRING(labels, ", ") as labels,
+    urgency
+FROM
+    @EVALUATION_TABLE
+"""
+
+query_all_feedback = """
+WITH CTE AS (
+  SELECT feedback_record_id,
+  STRING_AGG(response_value, ', ') AS response_value_full
+FROM
+  `govuk-ai-publishing.joined_feedback_view.feedback_6mth_with_metadata_updated`
```

Comment:
I don't think it's super important but should the table names be env variable names in this query string?

Reply:
This was an oversight. Fixed, good spot.